packages/quantic/playwright.config.ts

```diff
@@ -23,7 +23,9 @@ export default defineConfig({
   /* Opt out of parallel tests on CI. */
   workers: process.env.CI ? 4 : undefined,
   /* Reporter to use. See https://playwright.dev/docs/test-reporters */
-  reporter: process.env.CI ? [['list'], ['github']] : [['html'], ['list']],
+  reporter: process.env.CI
+    ? [['list'], ['github'], ['blob']]
```

Comment:
https://playwright.dev/docs/test-reporters#github-actions-annotations
You should not use the GitHub reporter with/ the matrix strategy